### ISSUE TITLE
Unable to run lm-scorer

### ISSUE THREAD

**Issue by abumafrim** [simonepri/lm-scorer#1]
Running the cli command

**!lm-scorer -lp --cuda 0 sentences.txt**,

I got the following error:

2020-04-11 07:48:41.170455: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
Error: __init__() got an unexpected keyword argument 'device'

and without **--cuda**, **!lm-scorer -lp sentences.txt**,

I got the following error

2020-04-11 07:52:48.219135: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
Error: Device index must be non-negative, got -1

Please help.

**Comment by simonepri** [comment]
Thanks for reporting fixed in [v0.2.1](https://github.com/simonepri/lm-scorer/releases/tag/v0.2.1).

**Comment by simonepri** [comment]
Thanks for reporting fixed in [v0.2.2](https://github.com/simonepri/lm-scorer/releases/tag/v0.2.2).

**Comment by PVinay737** [comment]
@simonepri , unable to install this package even with `--no-cache-dir` in pip. Any possible reasons ? 


```
Exception:
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/lib/python3.6/site-packages/pip/commands/install.py", line 346, in run
    requirement_set.prepare_files(finder)
  File "/usr/lib/python3.6/site-packages/pip/req/req_set.py", line 381, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/usr/lib/python3.6/site-packages/pip/req/req_set.py", line 623, in _prepare_file
    session=self.session, hashes=hashes)
  File "/usr/lib/python3.6/site-packages/pip/download.py", line 821, in unpack_url
    hashes=hashes
  File "/usr/lib/python3.6/site-packages/pip/download.py", line 663, in unpack_http_url
    unpack_file(from_path, location, content_type, link)
  File "/usr/lib/python3.6/site-packages/pip/utils/__init__.py", line 609, in unpack_file
    flatten=not filename.endswith('.whl')
  File "/usr/lib/python3.6/site-packages/pip/utils/__init__.py", line 498, in unzip_file
    data = zip.read(name)
  File "/usr/lib64/python3.6/zipfile.py", line 1338, in read
    return fp.read()
  File "/usr/lib64/python3.6/zipfile.py", line 858, in read
    buf += self._read1(self.MAX_N)
  File "/usr/lib64/python3.6/zipfile.py", line 948, in _read1
    data = self._decompressor.decompress(data, n)
MemoryError
```
